mainWrapper/src/main/java/com/cleanroommc/relauncher/wrapper/RelaunchMainWrapperV3.java

```diff
@@ -0,0 +1,27 @@
+package com.cleanroommc.relauncher.wrapper;
+
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+
+public class RelaunchMainWrapperV3 {
+    public static void main(String[] args) throws Throwable {
+        String mainClassName = System.getProperty("cleanroom.mainClass");
```

Comment:
We can standardize the relauncher property names, let's name this `cleanroom.relauncher.mainClass`